pkg/http/api.go

```diff
@@ -231,6 +231,34 @@ func SetSecrets(host string, verifyTLS bool, apiKey string, project string, conf
 	return computed, Error{}
 }
 
+// SetSecretNote for specified project and config
+func SetSecretNote(host string, verifyTLS bool, apiKey string, project string, config string, secret string, note string) (models.SecretNote, Error) {
+	reqBody := map[string]interface{}{}
+	reqBody["secret"] = secret
+	reqBody["note"] = note
+	body, err := json.Marshal(reqBody)
```

Comment:
Do we want to type this struct fully instead of using `map[string]interface{}{}`?